src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/MethodGCInfoNode.cs

```diff
@@ -164,6 +196,64 @@ private IEnumerable<GCInfoComponent> EncodeDataCore(NodeFactory factory)
                     yield return new GCInfoComponent(_methodNode.GCInfo);
                 }
             }
+#if READYTORUN
+            if (_methodNode.GetColdCodeNode() != null)
+            {
+                for (int frameInfoIndex = 0; frameInfoIndex < _methodNode.ColdFrameInfos.Length; frameInfoIndex++)
+                {
+                    byte[] unwindInfo = _methodNode.ColdFrameInfos[frameInfoIndex].BlobData;
+
+                    if (unwindInfo == null)
+                    {
+                        // Chain unwind info
+                        byte[] header = new byte[4];
+                        int i = 0;
+                        header[i++] = 1 + (4 << 3); // Version = 1, UNW_FLAG_CHAININFO
```

Comment:
Nit - at line 167 we already define constants for <code>UNW_FLAG_EHANDLER</code>, <code>UNW_FLAG_UHANDLER</code> and <code>FlagShift</code>, please consider moving them to the top of the method <code>EncodeDataCore</core> or making them static fields of <code>MethodGCInfoNode</code>, adding another entry for <code>UNW_FLAG_CHAININFO</code> and using the symbolic constants for construction of the header byte.

Reply:
Please see my comment below - we apparently already have the constants available, we should scrap our local duplicates and replace them with the framework enums.